app/views/statistics/index.html.slim

```diff
@@ -1,3 +1,6 @@
+css: 
+  .container {max-width: 1260px !important;}
+
```

Comment:
.container is declared in app/views/layouts/application.html.slim line:22
but I had to increase its max-width only on the /statistics page